src/lib/expert_partitioner/tree_views/partition.rb

```diff
@@ -37,7 +37,7 @@ def initialize(partition)
 
     def create
       tmp = ["Name: #{@partition.name}",
-             "Size: #{::Storage.byte_to_humanstring(@partition.size, false, 2, false)}"]
+             "Size: #{::Storage.byte_to_humanstring(@partition.size.to_i, false, 2, false)}"]
```

Comment:
`@partition.size` is a DiskSize object. You don't need to convert it back to integer and then use a libstorage function to format it. You can simply do `@partition.size.to_human_string`.

See http://www.rubydoc.info/github/yast/yast-storage-ng/master/Y2Storage/DiskSize